routing/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "routing"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[features]
+auto-learn = []
+
+[dependencies]
+ipnet = "2.11.0"
+iptrie = "0.9.1"
+mio = { workspace = true, features = ["os-ext", "net"] }
+net = { workspace = true }
+thiserror = { workspace = true }
+tracing = { workspace = true }
+tracing-subscriber = { workspace = true }
+dplane-rpc = { git = "https://github.com/githedgehog/dplane-rpc.git", branch = "master" }
```

Comment:
```
[...]
Caused by:
  Unable to update https://github.com/githedgehog/dplane-rpc.git?branch=master#d3547d38

Caused by:
  failed to fetch into: /home/runner/.cargo/git/db/dplane-rpc-c044331708d83dd5

Caused by:
  revision d3547d38735804f27d88dfc29db0c1985c5871c5 not found

Caused by:
  failed to authenticate when downloading repository
[...]
```

You'll need to make your repository public for CI to pass.

Reply:
Yes, I know. I forgot to do that.